src/main/java/io/confluent/examples/streams/WikipediaFeedAvroExample.java

```diff
@@ -129,6 +130,8 @@ static KafkaStreams buildWikipediaFeed(final String bootstrapServers,
     // in order to keep this example interactive.
     streamsConfiguration.put(StreamsConfig.COMMIT_INTERVAL_MS_CONFIG, 10 * 1000);
 
+    MonitoringInterceptorUtils.maybeConfigureInterceptors(streamsConfiguration);
```

Comment:
We might want to add a comment to explain this a little better such that people are not enforced to double check `MonitoringInterceptorUtils` itself